src/analysisd/analysisd.c

```diff
@@ -1848,7 +1848,7 @@ void * w_writer_log_thread(__attribute__((unused)) void * args ){
     #ifdef PRELUDE_OUTPUT_ENABLED
                 /* Log to prelude */
                 if (Config.prelude) {
-                    if (Config.prelude_log_level <= currently_rule->level) {
+                    if (Config.prelude_log_level <= lf->currently_rule->level) {
```

Comment:
This won't compile. The `Eventinfo `struct doesn't have a member called `currently_rule`. It's called `generated_rule` instead. This should be:

```
if (Config.prelude_log_level <= lf->generated_rule->level) {
```

Reply:
My bad, fixing that rn.